charcoal/compare_taxonomy.py

```diff
@@ -119,11 +123,11 @@ def choose_genome_lineage(guessed_genome_lineage, provided_lineage, match_rank,
 
 def get_genome_taxonomy(matches_filename, genome_sig_filename, provided_lineage,
                         tax_assign, match_rank, min_f_ident, min_f_major):
-    with open(matches_filename, 'rt') as fp:
-        try:
-            siglist = list(sourmash.load_signatures(fp, do_raise=True, quiet=True))
-        except sourmash.exceptions.SourmashError:
-            siglist = None
+    try:
+        siglist = list(sourmash.load_file_as_signatures(matches_filename))
+    # TR for empty signatures; update when #1537 is fixed. 
```

Comment:
suggest changing comment to be 'sourmash#1537' so it's clear it's in the sourmash issue tracker.